libraries/rush-sdk/src/index.ts

```diff
@@ -61,8 +63,48 @@ if (rushLibModule === undefined) {
 
 // SCENARIO 3:  A tool or script depends on "rush-sdk", and is meant to be used inside a monorepo folder.
 // In this case, we can use install-run-rush.js to obtain the appropriate rush-lib version for the monorepo.
-//
-// NOT IMPLEMENTED YET
+if (rushLibModule === undefined) {
+  try {
+    const rushJsonPath: string | undefined = tryFindRushJsonLocation(process.cwd());
+    if (!rushJsonPath) {
+      throw new Error('Could not find rush.json');
+    }
+    const monorepoRoot: string = path.dirname(rushJsonPath);
+
+    const rushJson: JsonObject = JsonFile.load(rushJsonPath);
+    const { rushVersion } = rushJson;
+
+    const installRunNodeModuleFolder: string = path.join(
+      monorepoRoot,
+      `common/temp/install-run/@microsoft+rush@${rushVersion}`
+    );
+
+    try {
+      // First, try to load the version of "rush-lib" that was installed by install-run-rush.js
+      rushLibModule = requireRushLibUnderFolderPath(installRunNodeModuleFolder);
```

Comment:
Ideally we should have a switch (e.g. `RUSH_SDK_DEBUG=1` environment variable) that enables verbose logging of each step of this algorithm, for example if it is failing on a CI machine where we can't just attach a debugger.

However I think it's outside the scope of this PR. It should be a separate PR.